created/json/itemlist_gison.json

```diff
@@ -5,7 +5,7 @@
         "name":"Gison's mushroom soup",
         "displaytype":"rare",
         "category":"food",
-        "description":"Eine hei�e, lecker riechende K�stlichkeit.",
+        "description":"Eine hei�e, lecker riechende K�stlichkeit.",
```

Comment:
It seems my text editor damaged the character encoding on this file...sorry about that!